src/server/gameserver/skill/SimpleTileCureSkill.cpp

```diff
@@ -768,9 +770,9 @@ void SimpleTileCureSkill::execute(Slayer* pSlayer, ZoneCoord_t X, ZoneCoord_t Y,
 			pPlayer->sendPacket(&_GCSkillToTileOK1);
 
 			//cout << "Healed Creature Size : " << (int)cList.size() << endl;
-    	    cList.push_back(pSlayer);
```

Comment:
This line is removed?

Reply:
This is not a correct fix... 
By not adding pSlayer to the cList, it avoid entering the for-loop,  then avoid the panic accidentally

Reply:
The only error is that  "}" is missing at 806.
it make clist at for-loop be change/,

771 delete pslayer, it can be ignored 
it be My server change/

Because my record was lost, it was submitted from my server file

Reply:
Oh, I see, the `cList` is moved out of the loop...